src/components/ShowPushNotificationTxDetails.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { pushCleanTxDetails } from '../actions';
+import PushTxDetailsModal from './PushTxDetailsModal';
+
+export default function ShowPushNotificationTxDetails() {
+  const txDetails = useSelector((state) => state.pushNotification.txDetails);
+  const dispatch = useDispatch();
+
+  if (!txDetails) return null;
+  return (
+    <PushTxDetailsModal
+      tx={txDetails.tx}
+      tokens={txDetails.tokens}
+      onRequestClose={() => dispatch(pushCleanTxDetails())}
+      isNFT={false}
```

Comment:
Why do you need this? It's not used by the component.

Reply:
Actually you should use this in the component, to show the balance of the token correctly. But not as a hardcoded value for the whole tx, you must use it for each token